src/main/java/frc/robot/RobotContainer.java

```diff
@@ -109,15 +109,16 @@ public class RobotContainer {
    * ***********************
    */
 
-  private SendableChooser<Command> autoChooser;
-
   /* The CommandXboxController instance must be static to allow the getter methods for its axes
    * to work.
    */
   /** Xbox controller for the driver. */
   public static CommandXboxController driverController =
       new CommandXboxController(Constants.Controllers.DRIVER_CONTROLLER_PORT);
 
+  /** Sendable Chooser for path planning autos. */
```

Comment:
"SendableChooser for autos" - this won't be *just* PathPlanner autos, we'll add our own as well.